### PR TITLE
Remove refs provided by Microsoft.NETCore.App.Ref targeting pack

### PR DESCRIPTION
**Pull request by RussKie**
Relates to https://github.com/dotnet/runtime/pull/54147
Relates to https://github.com/dotnet/windowsdesktop/pull/1747

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/dotnet/winforms/pull/5127)

### CODE REVIEW

**Review by dreddy-work — APPROVED** [review]

**Comment by ViktorHofer** [comment]
Microsoft.Win32.Registry.AccessControl isn't part of the Microsoft.NETCore.App.Ref targeting pack.

**Comment by RussKie** [comment]
Opps… Thank you

File: global.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
     }
   },
   "sdk": {
-    "version": "6.0.100-preview.4.21255.9"
+    "version": "6.0.100-preview.6.21317.3"
```
Comment (ViktorHofer) [inline global.json:14]: Wondering why the sdk update was necessary? Isn't the runtime flowing through the `VSRedistCommonNetCoreSharedFrameworkx6460PackageVersion` variable?

**Comment by ViktorHofer** [comment]
> Opps… Thank you

Apparently you don't use `Microsoft.Win32.Registry.AccessControl` anywhere so you can probably keep the drop of the dependency. But please double check yourself. Also can you please update the Versions.props file to remove the versions that aren't used anymore? Thanks

File: global.json (hunk 1 above)
Comment (RussKie) [reply global.json:14]: I don't believe it does. IIRC this is something that repos must do manually.

**Pull request merged**